lib/sorcery/model/temporary_token.rb

```diff
@@ -12,7 +12,7 @@ def self.included(base)
 
       # Random code, used for salt and temporary tokens.
       def self.generate_random_token
-        SecureRandom.urlsafe_base64(15).tr('lIO0', 'sxyz')
+        SecureRandom.urlsafe_base64(Controller::Config.token_randomness).tr('lIO0', 'sxyz')
```

Comment:
and there may be one more thing, you can calculate real length of the result value if you want.

```ruby
real_length = (15 * 3) / 4
SecureRandom.urlsafe_base64(real_length).tr('lIO0', 'sxyz').size
=> 15
```